cev_eris.dme

```diff
@@ -2820,6 +2828,7 @@
 #include "zzzz_modular_occulus\code\game\objects\items\plush.dm"
 #include "zzzz_modular_occulus\code\game\objects\items\devices\magnetic_lock.dm"
 #include "zzzz_modular_occulus\code\game\objects\items\devices\traitor_devices.dm"
+#include "zzzz_modular_occulus\code\game\objects\items\tools\_tools.dm"

```

Comment:
REEEEEEEE

Reply:
Those are our modular overrides. Somewhere in our history this got unchecked when it shouldn't be.